toutvcli/app.py

```diff
@@ -222,6 +222,8 @@ def _build_argparser(self):
         pf.add_argument('-q', '--quality', action='store',
                         default=App.QUALITY_AVG, choices=quality_choices,
                         help='Video quality (default: {})'.format(App.QUALITY_AVG))
+        pf.add_argument('-v', '--verbose', action='store_true',
```

Comment:
Didn't know that would work. Will remove all `--verbose` param handlers I added to pf/pi/pl. Thanks.